react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx

```diff
@@ -43,18 +45,26 @@ const isManifestValid = (manifest: Manifest): boolean =>
       manifest.theme_color
   )
 
-type Props = ManifestData & InjectedIntlProps & MutationProps & QueryResult
+type Props = PWAData & InjectedIntlProps & MutationProps & QueryResult
 
 const PWAForm: React.FunctionComponent<Props> = ({
   manifest: pwaManifest,
   splashes,
   iOSIcons,
+  pwaSettings,
   updateManifest,
   updateManifestIcon,
+  updatePWASettings,
   intl,
   refetch,
 }) => {
-  const [manifest, setManifest] = useState(fillManifest(pwaManifest))
+  const [manifest, setManifest]: [Manifest, (m: Manifest) => void] = useState(
```

Comment:
Why doesn't `fillManifest` return Manifest? Typesctirp automatically infers the typings in setState, as long as you provide a typed default